pkg/kubectl/.import-restrictions

```diff
@@ -1,151 +1,130 @@
-                "k8s.io/kubernetes/pkg/client/unversioned",
-                "k8s.io/kubernetes/pkg/cloudprovider",
-                "k8s.io/kubernetes/pkg/cloudprovider/providers/aws",
-                "k8s.io/kubernetes/pkg/controller",
-                "k8s.io/kubernetes/pkg/controller/daemon",
-                "k8s.io/kubernetes/pkg/controller/daemon/util",
-                "k8s.io/kubernetes/pkg/controller/deployment/util",
-                "k8s.io/kubernetes/pkg/controller/history",
-                "k8s.io/kubernetes/pkg/controller/statefulset",
-                "k8s.io/kubernetes/pkg/credentialprovider",
-                "k8s.io/kubernetes/pkg/credentialprovider/aws",
-                "k8s.io/kubernetes/pkg/features",
-                "k8s.io/kubernetes/pkg/fieldpath",
-                "k8s.io/kubernetes/pkg/generated",
-                "k8s.io/kubernetes/pkg/kubectl",
-                "k8s.io/kubernetes/pkg/kubelet/apis",
-                "k8s.io/kubernetes/pkg/kubelet/qos",
-                "k8s.io/kubernetes/pkg/kubelet/types",
-                "k8s.io/kubernetes/pkg/master/ports",
-                "k8s.io/kubernetes/pkg/printers",
-                "k8s.io/kubernetes/pkg/printers/internalversion",
-                "k8s.io/kubernetes/pkg/registry/rbac/reconciliation",
-                "k8s.io/kubernetes/pkg/registry/rbac/validation",
-                "k8s.io/kubernetes/pkg/scheduler/algorithm",
-                "k8s.io/kubernetes/pkg/scheduler/algorithm/predicates",
-                "k8s.io/kubernetes/pkg/scheduler/algorithm/priorities/util",
-                "k8s.io/kubernetes/pkg/scheduler/api",
-                "k8s.io/kubernetes/pkg/scheduler/cache",
-                "k8s.io/kubernetes/pkg/scheduler/util",
-                "k8s.io/kubernetes/pkg/scheduler/volumebinder",
-                "k8s.io/kubernetes/pkg/security/apparmor",
-                "k8s.io/kubernetes/pkg/serviceaccount",
-                "k8s.io/kubernetes/pkg/util/file",
-                "k8s.io/kubernetes/pkg/util/goroutinemap",
-                "k8s.io/kubernetes/pkg/util/hash",
-                "k8s.io/kubernetes/pkg/util/interrupt",
-                "k8s.io/kubernetes/pkg/util/io",
-                "k8s.io/kubernetes/pkg/util/labels",
-                "k8s.io/kubernetes/pkg/util/metrics",
-                "k8s.io/kubernetes/pkg/util/mount",
-                "k8s.io/kubernetes/pkg/util/net/sets",
-                "k8s.io/kubernetes/pkg/util/node",
-                "k8s.io/kubernetes/pkg/util/nsenter",
-                "k8s.io/kubernetes/pkg/util/parsers",
-                "k8s.io/kubernetes/pkg/util/slice",
-                "k8s.io/kubernetes/pkg/util/taints",
-                "k8s.io/kubernetes/pkg/version",
-                "k8s.io/kubernetes/pkg/version/prometheus",
-                "k8s.io/kubernetes/pkg/volume",
-                "k8s.io/kubernetes/pkg/volume/util",
-                "k8s.io/utils/pointer"
+			"k8s.io/kubernetes/pkg/api",
```

Comment:
hm, any idea why the new diff uses tabs instead of spaces for the indentation?

Reply:
fixed now.